src/input_xml.F90

```diff
@@ -5856,7 +5813,8 @@ subroutine read_ce_cross_sections(libraries, library_dict)
           ! Read nuclide data from HDF5
           file_id = file_open(libraries(i_library) % path, 'r')
           group_id = open_group(file_id, name)
-          call nuclides(i_nuclide) % from_hdf5(group_id)
+          call nuclides(i_nuclide) % from_hdf5(group_id, &
+                                               materials(i) % temperature)
```

Comment:
No you're right.  This was an oversight.  I'm trying to think of the best way to fix this.  If I can't fix it without a substantial change that wont persist after we switch to the `Nuclide` class containing all relevant temperature information, then I will just plow forward to get the T dependence within the `Nuclide` class now in this PR.